### PR TITLE
add feed for cascoda ca8210 driver

### PR DESCRIPTION
**Pull request by abhijit-mahajani-imgtec**
Also merged master into dev for missed u-boot commit.

### CODE REVIEW

File: cascoda/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=cascoda
```
Comment (pratik-prajapati-imgtec) [inline cascoda/Makefile]: this should be cascoda-ca8210

File: cascoda/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  DEPENDS:=@TARGET_pistachio_marduk +kmod-mac802154 +kmod-ieee802154_6lowpan \
+           +kmod-nhc_udp +kmod-nhc_fragment +kmod-nhc_dest +kmod-nhc_hop \
+           +kmod-nhc_mobility +kmod-nhc_routing +kmod-nhc_ipv6
+  FILES:=$(PKG_BUILD_DIR)/Module/ca8210sm.ko
+  AUTOLOAD:=$(call AutoLoad,92,ca8210sm)
+  KCONFIG:=
+endef
+
+define KernelPackage/cascoda/description
+ Cascoda ca8210 Driver
```
Comment (pratik-prajapati-imgtec) [inline cascoda/Makefile]: one more space

File: cascoda/Makefile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+PKG_SOURCE_URL:=https://github.com/FlowM2M/ca821x.git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define KernelPackage/cascoda
+  SUBMENU:=WPAN 802.15.4 Support
+  TITLE:=CASCODA CA821x Driver
```
Comment (nikhil-zinjurde-imgtec) [inline cascoda/Makefile]: should be CA8210

**Comment by nikhil-zinjurde-imgtec** [comment]
Please also update the readme.md to include uboot and ca8210 driver description

**Comment by pratik-prajapati-imgtec** [comment]
Approved and Verified

**Pull request merged**